samples/WinFormsDx12/WinFormsDx12.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
```

Comment:
Same here, but the header should be:
```
<?xml version="1.0" encoding="utf-8"?>
<!-- Copyright © Tanner Gooding and Contributors. Licensed under the MIT License (MIT). See License.md in the repository root for more information. -->
```

Reply:
added copyright notices to headers and a little more code cleanup.